### PR TITLE
redirect for install/gui

### PR DESCRIPTION
**Pull request by joel-hamill**

### CODE REVIEW

File: redirects [hunk 1]
```diff
@@ -18,3 +18,5 @@
 /overview/components/index.html /docs/latest/overview/components/
 /support/index.html /community/
 /usage/cli/install/index.html /docs/latest/usage/cli/install/
+/administration/installing/custom/gui/ /docs/latest/administration/installing/custom/gui/
```
Comment (pyronicide) [inline redirects]: You'll need to include `index.html` at the end of the sources.

Comment (joel-hamill) [inline redirects]: @pyronicide done

**Comment by karlkfi** [comment]
The blank lines on the end of the redirect file broke develop's `update-redirects.sh`

**Comment by karlkfi** [comment]
Fixed: https://github.com/dcos/dcos-website/commit/182e79c58d21452b99c5d6386585a7db33df68a1 - http://velocity.mesosphere.com/service/velocity/view/DCOS%20Website/job/public-dcos-website-deploy-dev/411/

**Pull request merged**